src/components/tab/EmptyContent.vue

```diff
@@ -15,7 +15,8 @@
 				</div>
 			</div>
 			<div v-if="showConnectButton" class="empty-content--connect-button">
-				<OAuthConnectButton :request-url="requestUrl" />
+				<OAuthConnectButton :request-url="requestUrl"
+					:file-info="fileInfo" />
```

Comment:
```suggestion
				<OAuthConnectButton :request-url="requestUrl" :file-info="fileInfo" />
```
The indent looks a little bit odd

Reply:
I've put it all in one line, actually its not very long